lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -219,6 +219,14 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
         return this._tagNameControl;
     }
 
+    get isNameTagsVisible(): boolean {
+        return (!this.tagNameControlVisible && this.tags?.length > 0) || this.tagNameControlVisible;
+    }
+
+    get isTagsEmpty(): boolean {
```

Comment:
this code returns true when there are tags present, how do you expect it to be Empty ?